### PR TITLE
TG-24: Time sheets model

### PR DESCRIPTION
**Pull request by GonzaloBracamonte**

### CODE REVIEW

File: src/models/Time-sheets.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const { Schema } = mongoose;
+
+const timeSheetsSchema = new Schema({
+  id: {
+    type: Number,
+    required: true,
+  },
+  employeeId: {
+    type: String,
+    required: true,
```
Comment (Javier-Cervera) [inline src/models/Time-sheets.js:12]: Maybe we could use `type: mongoose.Schema.Types.ObjectId`, and next week add `ref: 'Employees'`

File: src/models/Time-sheets.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  project: {
+    type: String,
+    required: true,
+  },
+  date: {
+    type: Date,
+    required: true,
+  },
+  hours: {
+    type: String,
```
[the remark this answers is not in the document]
Comment (GonzaloBracamonte) [reply src/models/Time-sheets.js:27]: It could be, now I will change it

File: src/models/Time-sheets.js (hunk 1 above)
Comment (GonzaloBracamonte) [reply src/models/Time-sheets.js:12]: You´re right, i will change this too

File: src/models/Time-sheets.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const timeSheetsSchema = new Schema({
+  employeeId: {
+    type: mongoose.Schema.Types.ObjectId,
```
Comment (AlfonsoDalix) [inline src/models/Time-sheets.js:7]: Por ahora dejemoslo como un String, ya que no vamos a relacionarlos

**Review by AlfonsoDalix — CHANGES_REQUESTED** [review]

File: src/models/Time-sheets.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    required: true,
+  },
+  approved: {
+    type: Boolean,
+    required: true,
+  },
+  role: {
+    type: String,
+    required: true,
+  },
```
Comment (Javier-Cervera) [inline src/models/Time-sheets.js:37]: We could add a property `enum: ['DEV', 'QA', 'PM', 'TL']`

File: src/models/Time-sheets.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  task: {
+    type: String,
+    required: true,
+  },
+  approved: {
+    type: Boolean,
+    required: true,
+  },
+  role: {
```
Comment (AlfonsoDalix) [inline src/models/Time-sheets.js:34]: role te tendría que quedar así
role: {
    type: String,
    enum: ['DEV', 'QA', 'PM', 'TL'],
    required: true,
}

**Review by AlfonsoDalix — APPROVED** [review]

**Pull request merged**